pkg/madmin/README.md

```diff
@@ -262,6 +262,17 @@ Fetches Mem utilization for all cluster nodes. Returned value is in Bytes.
 | `mem.Usage.Mem`   | _uint64_ | The total number of bytes obtained from the OS         |
 | `mem.Usage.Error` | _string_ | Error (if any) encountered while accesing the CPU info |
 
+<a name="NetPerfInfo"></a>
+### NetPerfInfo() (map[string][]NetPerfInfo, error)
+
+Fetches network performance of all cluster nodes. Returned value is in duration in nanosecond count for each node.
```

Comment:
IIUC, `ReadPerf` is measured in the duration (in ns) to read `100MiB` by default. This is not obvious to the `madmin` package consumer. Also, doesn't it make sense to take the payload size as a parameter to `NetPerfInfo`?

Reply:
ReST API supports optional size. In golang there is no optional param support. Adding here as mandatory argument.